src/formats/toml.js

```diff
@@ -4,13 +4,19 @@ import moment from 'moment';
 import AssetProxy from 'ValueObjects/AssetProxy';
 import { sortKeys } from './helpers';
 
+// IE polyfill for Number.isInteger
+Number.isInteger = Number.isInteger || (value => typeof value === "number" && isFinite(value) && Math.floor(value) === value);
+
 const outputReplacer = (key, value) => {
   if (moment.isMoment(value)) {
     return value.format(value._f);
   }
   if (value instanceof AssetProxy) {
     return `${ value.path }`;
   }
+  if (Number.isInteger(value)) {
```

Comment:
I wonder if we should be checking `isSafeInteger` instead, so we don't accidentally output an invalid number? Or is that only a concern for computations and it shouldn't be a problem with `toString()`? That was the original problem that @iology noted for `tomlify` anyway.

Reply:
I think `isInteger` is sufficient... we already know we aren't dealing with a string representation of an `int`-like `number` that could overflow or lose precision. We're dealing with a straight-up `number` that JavaScript believes to be an `int`. And right, no calculations here. We're just telling `tomlify` not to put decimal places on the thing, which we must do by giving it a string representation of what we want rendered in the TOML file.